denylist/src/denylist.rs

```diff
@@ -23,6 +19,7 @@ pub struct DenyList {
     pub client: DenyListClient,
     #[serde(skip_serializing)]
     pub filter: Xor32,
+    pub valid_sign_keys: Vec<PublicKey>,
```

Comment:
any significance to `valid_` in `valid_sign_keys`? seems like any configured signing key would be expected to be valid

Reply:
Renamed 0b9db6e